Index: ZasasMagicMacros.toc
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>## Interface: 11200\r\n## Title: ZasasMagicMacros\r\n## Author: Zasamel\r\n## Version: 1.0\r\n## Notes: Magical Macros!\r\n\r\n# === Data ===\r\nsrc\\utils\\Rotations.lua\r\n\r\n# === Utils ===\r\nsrc\\utils\\AbilityNamesToTexturesMapping.lua\r\nsrc\\utils\\Buffs.lua\r\nsrc\\utils\\Combat.lua\r\nsrc\\utils\\Config.lua\r\nsrc\\utils\\Debuffs.lua\r\nsrc\\utils\\Abilities.lua\r\nsrc\\utils\\RotationRunner.lua\r\nsrc\\utils\\Setup.lua\r\nsrc\\utils\\Slots.lua\r\nsrc\\utils\\ZMM_MacroBuilder.lua\r\nsrc\\utils\\Setup.lua\r\n\r\n# === Paladin ===\r\nsrc\\classes\\paladin\\Core.lua\r\nsrc\\classes\\paladin\\OneButtonProtection.lua\r\nsrc\\classes\\paladin\\OneButtonProtectionAoE.lua\r\nsrc\\classes\\paladin\\SealOfRighteousness.lua\r\nsrc\\classes\\paladin\\SealOfWisdom.lua\r\nsrc\\classes\\paladin\\SmartCrusaderStrike.lua\r\n\r\n# === Frames ===\r\nsrc\\ZMM_Main.lua
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/ZasasMagicMacros.toc b/ZasasMagicMacros.toc
--- a/ZasasMagicMacros.toc	(revision 5852bf27bea1c3ab9dd2a8bcbdc6558b92c54975)
+++ b/ZasasMagicMacros.toc	(date 1764337266638)
@@ -18,7 +18,10 @@
 src\utils\Setup.lua
 src\utils\Slots.lua
 src\utils\ZMM_MacroBuilder.lua
-src\utils\Setup.lua
+
+# === Hunter ===
+src\classes\hunter\Core.lua
+src\classes\hunter\OneButtonHunterPetPullReset.lua
 
 # === Paladin ===
 src\classes\paladin\Core.lua
Index: src/classes/hunter/Core.lua
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/classes/hunter/Core.lua b/src/classes/hunter/Core.lua
new file mode 100644
--- /dev/null	(date 1764337048077)
+++ b/src/classes/hunter/Core.lua	(date 1764337048077)
@@ -0,0 +1,2 @@
+Hunter = Hunter or {}
+Hunter.__index = Hunter
\ No newline at end of file
Index: src/classes/hunter/OneButtonHunterPetPullReset.lua
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/classes/hunter/OneButtonHunterPetPullReset.lua b/src/classes/hunter/OneButtonHunterPetPullReset.lua
new file mode 100644
--- /dev/null	(date 1764337063406)
+++ b/src/classes/hunter/OneButtonHunterPetPullReset.lua	(date 1764337063406)
@@ -0,0 +1,11 @@
+function Hunter:OneButtonHunterPetPullReset()
+    CastSpellByName("Furious Howl")
+    if UnitExists("pet") then
+        if Buffs:findBuffByTextureName("Ability_Rogue_FeignDeath") then
+            CastSpellByName("Eyes of the Beast")
+        end
+        CastSpellByName("Feign Death")
+    end
+    CastSpellByName("Call Pet")
+    CastSpellByName("Revive Pet")
+end
\ No newline at end of file
